app/controllers/api/base_controller/manager.rb

```diff
@@ -140,7 +140,7 @@ def update_multiple_collections(is_subcollection, target, type, resources)
           update_one_collection(is_subcollection, target, type, rid, r)
         end
         raise BadRequestError, "No #{type} resources were specified for the #{action} action" if processed == 0
-        {"results" => results}
+        {"results" => results.flatten}
```

Comment:
I just double checked my logic just to be sure and am not sure my explanation above made sense, so here is the full results if the flatten is moved up:
```
{"results"=>
  [[{"success"=>true,
     "message"=>"Assigning Tag: category:'department' name:'finance'",
     "href"=>"http://www.example.com/api/vms/",
     "tag_category"=>"department",
     "tag_name"=>"finance",
     "tag_href"=>"http://www.example.com/api/tags/10000000002666"}],
   [{"success"=>true,
     "message"=>"Assigning Tag: category:'cc' name:'001'",
     "href"=>"http://www.example.com/api/vms/",
     "tag_category"=>"cc",
     "tag_name"=>"001",
     "tag_href"=>"http://www.example.com/api/tags/10000000002670"}]]}
```